src/pages/home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+//requisito 2
+
+class Home extends React.Component {
+  render () {
```

Comment:
Correção apontada pelo ESLint
```suggestion
  render() {
```